product-service/handlers/get-product-by-id/index.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import products from '../../mocks/products';
+
+export const handler: APIGatewayProxyHandler = async (event, _context) => {
```

Comment:
@anderysenuta Request to `/products/{id}` route suggests me downloading a `test.csv` file with the following content:

```csv
1,"blah",123
2,"qwe",456
```

No matter what `id` I type in, the response content is the same.

Could you fix it?